src/julia/magic.py

```diff
@@ -22,7 +22,7 @@
 import sys
 import warnings
 
-from IPython.core.magic import Magics, magics_class, line_cell_magic
+from IPython.core.magic import Magics, magics_class, line_cell_magic, no_var_expand
```

Comment:
Let's support older IPython with

```python
try:
    from IPython.core.magic import no_var_expand
except ImportError:
    def no_var_expand(f):
        return f
```

(put it after `from .tools import redirect_output_streams`)